local-cli/generator-ios/templates/app/AppDelegate.m

```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle"];
+  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios&dev=true"];
```

Comment:
Why is this needed? I thought `dev` is `true` by default, and `platform` can be inferred from `.ios` extension. @amasad ?